ext/mysqlnd/mysqlnd_alloc.c

```diff
@@ -60,10 +58,7 @@ PHPAPI const char * mysqlnd_debug_std_no_trace_funcs[] =
 	NULL /* must be always last */
 };
 
-#if MYSQLND_DEBUG_MEMORY
-
-
-#if ZEND_DEBUG
+#if PHP_DEBUG
 #else
 #define __zend_orig_filename "/unknown/unknown"
```

Comment:
Is this else necessary?

Reply:
I think this whole section needs to be dropped, thanks for pointing this out